balancer-labs/frontend-v2
# Sor v2.0.0 beta.4

# Description

This PR updates the SOR to the new beta version which removes `bignumber.js` from its interface (barring the `marketSp`). The SOR manager has been updated to match this new interface.

As part of this I've changed the SORManager to default to returning `ethers` rather than `bignumber.js` bignumbers which required the `useGnosis` composable to switch to preferring these as well.

## Type of change


## How should this be tested?

Check that SOR quotes match that on production site.
Check that Gnosis UI matches that on production site.

## Checklist:
